ci/pkl-pipelines/pay-deploy/concourse-runner.pkl

```diff
@@ -0,0 +1,251 @@
+amends "package://pkg.pkl-lang.org/github.com/alphagov/pkl-concourse-pipeline/pkl-concourse-pipeline@0.0.3#/Pipeline.pkl"
+
+import "package://pkg.pkl-lang.org/github.com/alphagov/pkl-concourse-pipeline/pkl-concourse-pipeline@0.0.3#/Pipeline.pkl"
+
+import "../common/pipeline_self_update.pkl"
+import "../common/shared_resources.pkl"
+
+resource_types {
+  shared_resources.pullRequestResourceType
+}
+
+resources = new {
```

Comment:
Also, need to add `pipeline_self_update.PayPipelineSelfUpdateResource("pay-deploy/concourse-runner.pkl", "master")`

Reply:
Urgh, his was something I forgot to come back too. There is another resource specified for this, but it's not looking at the pkl and I meant to confirm how that worked.